torch/nn/functional.py

```diff
@@ -6,6 +6,7 @@
 import torch
 from . import _functions
 from .modules import utils
+from .backends.thnn import backend as thnn_backend
```

Comment:
@apaszke I find this `backends.thnn` a bit confusing. Is this dead code or half-dead code? I suppose it is there to make room for other backends (like `nnpack` maybe?), but is it still the case? The only use I see (but I'm probably missing something) is that it [renames `FeatureDropout` to `Dropout2d` and `Dropout3d`](https://github.com/pytorch/pytorch/blob/master/torch/nn/backends/thnn.py#L40-L41).